e2e_test/batch/catalog/pg_size.slt.part

```diff
@@ -4,8 +4,6 @@ create table t (v1 int);
 statement ok
 insert into t values (3);
 
-sleep 30s
-
```

Comment:
@hzxa21 Are the sleeps necessary given we already enabled implicit flush?

Reply:
table size is propagated to frontend through async notification. I think a sleep is still needed since a successful flush doesn't mean the notification is received by frontend. But 30s sleep seems too long. We can make it 3s I think.